src/pages/pattern-library.tsx

```diff
@@ -88,6 +89,37 @@ function Page() {
             </ul>
           </section>
 
+          <section className="grid-section grid-content">
+            <h2 className="title-subsection">SKU Selectors</h2>
+            <ul className="list-vertical">
+              <li>
+                <SkuSelector
+                  label="Size"
+                  variant="size"
+                  options={[
+                    { label: 'S', value: 's' },
+                    { label: 'M', value: 'm' },
+                    { label: 'L', value: 'l' },
+                    { label: 'XL', value: 'xl', disabled: true },
+                  ]}
+                />
+              </li>
+              <li>
+                <SkuSelector
+                  label="Color"
+                  variant="color"
+                  options={[
+                    { label: 'Yellow', value: '#f1d096' },
+                    { label: 'Pink', value: '#eed0d0' },
+                    { label: 'Green', value: '#b2dbcb' },
+                    { label: 'Blue', value: '#bacbec' },
+                    { label: 'Lilac', value: '#ebdcff', disabled: true },
+                  ]}
+                />
+              </li>
```

Comment:
I miss the image variant. Not blocking

Reply:
It will only be styled after (for the Beta version)